### PR TITLE
Add promoted operators

### PR DESCRIPTION
**Pull request by vincenthongzy**
Two pages with slightly different information displayed:
1. `/operators`
2. `/avs/:address/operators`

Some edge cases to consider:
1. When we search for operators, we should always display the ads even if the result is empty.
2. In `/avs/:address/operators`, it is ok to not show promoted operators if they are not currently securing the AVS
3. If the promoted operators endpoint fails, we don't need to display error.

Backend PR:
https://github.com/NethermindEth/restaking-dashboard-backend/pull/114

### CODE REVIEW

**Review by Zishan-7 — CHANGES_REQUESTED** [review]
On `/operators` page when search result is empty, the promoted operators rows expand to take all of the available space

Also ETH -> USD conversion shows `NaN` when search result is empty. This happens on the AVS details page as well

Screenshot 2024-09-03 at 12 34 39 PM

**Comment by vincenthongzy** [comment]
> On `/operators` page when search result is empty, the promoted operators rows expand to take all of the available space
> 
> Also ETH -> USD conversion shows `NaN` when search result is empty. This happens on the AVS details page as well
> 
> Screenshot 2024-09-03 at 12 34 39 PM

Hm strange, the expanding rows on empty results should have been fixed

**Comment by vincenthongzy** [comment]
> On `/operators` page when search result is empty, the promoted operators rows expand to take all of the available space
> 
> Also ETH -> USD conversion shows `NaN` when search result is empty. This happens on the AVS details page as well
> 
> Screenshot 2024-09-03 at 12 34 39 PM

Fixed by decoupling the promoted operators' rate and normal operators' rate on both BE and FE

File: src/avs/AVSDetailsOperatorsTab.jsx [hunk 1]
```diff
@@ -210,6 +213,20 @@ function AVSOperatorsList({ address, avsError, isAVSLoading, tvl }) {
...
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await rdService.getAVSPromotedOperators(address);
+        dispatch({
+          promotedOperators: response.results,
+          promotedOperatorsRate: response.rate
+        });
+      } catch (e) {
+        //TODO: handle error
```
Comment (rubo) [inline src/avs/AVSDetailsOperatorsTab.jsx:225]: Why no error handling?

Comment (vincenthongzy) [reply src/avs/AVSDetailsOperatorsTab.jsx:225]: If the promoted operators endpoint fails, we don't need to display error, so we just swallow it up

**Review by Zishan-7 — APPROVED** [review]

File: src/avs/AVSDetailsOperatorsTab.jsx (hunk 1 above)
Comment (rubo) [reply src/avs/AVSDetailsOperatorsTab.jsx:225]: That `TODO` confused me.

**Review by rubo — APPROVED** [review]

File: src/avs/AVSDetailsOperatorsTab.jsx (hunk 1 above)
Comment (vincenthongzy) [reply src/avs/AVSDetailsOperatorsTab.jsx:225]: I'll reword this comment

**Pull request merged**